### ISSUE TITLE
Ipe crashes when opening a file with a singular matrix

### ISSUE THREAD

**Issue by LukasXperiaZ** [otfried/ipe#437]
I constructed something for my bachelor thesis and suddenly when I wanted to select something Ipe just closed. This is reproducable and the following construction (see file attached) should make Ipe close. I haven't found the reason for that yet though.
[BUG_closes_when_trying_to_select.zip](https://github.com/otfried/ipe-issues/files/9838069/BUG_closes_when_trying_to_select.zip)

**Comment by otfried** [comment]
Your figure contains an ellipse with radius zero:
```
<path matrix="1 0 0 1 0 -16" stroke="lightgray">
0 0 0 0 107.713 512 e
</path>
```
If you delete these three lines from the .ipe file, it will work.

Of course Ipe shouldn't just crash.  I wonder if it should report a parsing error, or just ignore the object.  (And I should check why it doesn't report an error when you create such an object.)

**Comment by otfried** [comment]
Fixed in 7.2.28.
